build.gradle

```diff
@@ -13,6 +13,7 @@ allprojects {
     repositories {
         jcenter()
         maven { url 'https://esri.bintray.com/arcgis' }
+        maven { url 'http://olympus.esri.com/artifactory/arcgisruntime-repo/' }
```

Comment:
I think this might confuse external users, we should keep it internal only.

Reply:
I put this in on @gunt0001 's suggestion to make it easier for testing with internal builds. We do it already for the toolkit.

https://github.com/Esri/arcgis-runtime-toolkit-android/blob/master/build.gradle#L38